src/verifact_manager.py

```
@@ -9,12 +9,16 @@
 and provides both synchronous and asynchronous operation modes.
 """
 
+# Configure OpenAI client to use OpenRouter
+from src.utils.openrouter_config import *
```

Ruff (0.11.9)

`from src.utils.openrouter_config import *` used; unable to detect undefined names

(F403)